src/systems/a320_systems/src/pneumatic.rs

```diff
@@ -104,6 +104,45 @@ valve_signal_implementation!(EngineStarterValveSignal);
 valve_signal_implementation!(FanAirValveSignal);
 valve_signal_implementation!(PackFlowValveSignal);
 
+fn hydraulic_reservoir_init_pressure_from_sim_state(context: &InitContext) -> Pressure {
+    // When starting with engine off we set pressure value close to nominal so that low pressure
+    // faults do not trigger
+    const COLD_INIT_PRESS_PSI: f64 = 45.;
+
+    // When starting with engine running we set lower pressure so we let bleed system converge to
+    // nominal value
+    const HOT_INIT_PRESS_PSI: f64 = 50.;
+
+    if context.has_engines_running() {
+        println!("HOT INIT 5");
+        Pressure::new::<psi>(HOT_INIT_PRESS_PSI)
+    } else {
+        println!("COLD INIT 45");
+        Pressure::new::<psi>(COLD_INIT_PRESS_PSI)
+    }
+}
+
+fn hydraulic_reservoir_init_level_from_sim_state(
+    context: &InitContext,
+    circuit_id: HydraulicColor,
+) -> Volume {
+    const HOT_INIT_LEVEL_OFFSET_GAL: f64 = 0.5;
+
+    let init_level_cold = match circuit_id {
+        HydraulicColor::Green => Volume::new::<gallon>(2.475960068),
+        HydraulicColor::Blue => Volume::new::<gallon>(1.081721769),
+        HydraulicColor::Yellow => Volume::new::<gallon>(1.683443537),
+    };
+
+    if context.has_engines_running() {
+        println!("HOT INIT LEVEL XX");
```

Comment:
I know this is still marked as draft, just noting that I don't think we should be logging for a non-test function like this and this should be removed before merging.

Reply:
Yeah of course, just added those when investigating why the quick and easy fix is just not working at all.....

Reverted to draft..... Might delete the PR as it seems it's a deeper issue to solve in bleed part